src/SummaryRow.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import SummaryCell from './SummaryCell';
+import { SummaryRendererProps } from './common/types';
+
+export default function SummaryRow<R>({
+  extraClasses,
+  height,
+  rowIdx,
+  lastFrozenColumnIndex,
+  row,
+  scrollLeft,
+  viewportColumns,
+  width
+}: SummaryRendererProps<R>) {
+  return (
+    <div
+      className={classNames(
```

Comment:
May be we can add `rdg-summary-row` class as well so it is easy to customize styles. wdyt?